tds.go

```diff
@@ -965,6 +966,27 @@ func parseConnectParams(dsn string) (connectParams, error) {
 		}
 	}
 
+	// https://msdn.microsoft.com/en-us/library/dd304019.aspx
```

Comment:
This link doesn't really have special information for this. This is just a portion of the TDS spec.